artifactory.py

```diff
@@ -64,6 +64,10 @@
 default_config_path = "~/.artifactory_python.cfg"
 global_config = None
 
+# set logger to be configurable from external
+log = logging.getLogger(__name__)
```

Comment:
let's use `logging.getLogger('artifactory')` explicitly, even in `dohq_artifactory`

Reply:
it is not the best practice. See for example how requests are configured: https://github.com/psf/requests/blob/1e5fad7433772b648fcbc921e2a79de5c4c6be8b/requests/__init__.py#L145

we strongly depend on requests, I would follow their structure. Also, after some research I see that literature recommends the same, eg David Beazley